yaml_files/cuba.yml

```diff
@@ -565,3 +555,59 @@ CUBA_KEYS:
     definition: Thermodynamic ensemble
     shape: [1]
     type: string
+
+  LATTICE_UC_ABC:
```

Comment:
Lattice vectors refers to the three 3D vectors that define the unit cell. Here we are not requesting these vectors but the lattice parameters (a,b,c and alpha, beta, gamma) that are the most common way to define the Unit Cell (even more common that the vectors), in particular in atomics builders. In our concern, it is much more easier to ask to the user for a set of 3 length parameters (a,b,c) and a set of three angle parameters (alpha, beta, gamma) than ask for the six parameters at the same time. If we use "Lattice parameters" we have to ask for 6 parameters. That is the reason we have add a "surname" to these two CUDAs. Anyway, name of the CUDA could be discussed, but we cannot change to Lattice Vectors. Besides, it is not so standard as Lattice Parameters